app/.server/event-web-api/event-api.test.ts

```diff
@@ -22,26 +22,38 @@ describe('#EventApi', () => {
         formats: [format],
         categories: [category],
         talk: await talkFactory({ speakers: [speaker], attributes: { level: 'BEGINNER', languages: ['fr'] } }),
+        traits: ['confirmed'],
       });
 
       const eventApi = new EventApi(event.slug, '123');
       const result = await eventApi.proposals({});
 
       expect(result).toEqual({
+        startDate: event.conferenceStart,
+        endDate: event.conferenceEnd,
         name: event.name,
         proposals: [
           {
             title: proposal.title,
             abstract: proposal.abstract,
             level: 'BEGINNER',
+            references: proposal.references,
             formats: [format.name],
+            reviews: [],
```

Comment:
It could be great to add a review using `reviewFactory`, to test the API output and be sure to not break it in the future